cmake/modules/FindLibBot.cmake

```diff
@@ -7,14 +7,21 @@
 # LIBBOT_INCLUDE_DIRS
 # LIBBOT_LIBRARIES
 
+find_package(GTK2 REQUIRED gtk)
+
+find_package(libbot QUIET)
```

Comment:
Ok.  Yeah I'm not thrilled about some of these recent changes.  I understand why they are necessary changes for you to get things to build with drake bazel.  My concern is that it introduces secondary code paths that are only executed when using a drake bazel build, so this code could break.  I'll get Spartan CI running soon, that will help.

Maybe a good eventual solution would be for the projects like libbot to be updated to produce libbot-config.cmake so then the findpackage can be unified, if that makes sense.